src/ray/gcs/gcs_server/gcs_autoscaler_state_manager.cc

```diff
@@ -184,13 +179,58 @@ void GcsAutoscalerStateManager::GetClusterResourceConstraints(
   }
 }
 
+void GcsAutoscalerStateManager::UpdateResourceLoadAndUsage(
+    const rpc::ResourcesData &data) {
+  NodeID node_id = NodeID::FromBinary(data.node_id());
+  auto iter = node_resource_info_.find(node_id);
+  if (iter == node_resource_info_.end()) {
+    iter = node_resource_info_
+               .emplace(node_id, std::make_pair(absl::Now(), rpc::ResourcesData()))
+               .first;
+  }
+
+  auto &new_data = iter->second.second;
+
+  if (data.resource_load_changed()) {
```

Comment:
Don't we always report the load?

Reply:
yeah, I guess it's safe to assume it changed